.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,8 @@
+name: Tag and Release
+on:
+  - issues
+  - pull_request
+jobs:
+  label-check:
+    runs-on: ubuntu-latest
+    steps: kimseungbin/tag-and-release@v1
```

actionlint

"steps" section must be sequence node but got scalar node with "!!str" tag

(syntax-check)